setup.py

```diff
@@ -58,6 +61,12 @@ def build_extension(self, ext):
             "-DPYTHON_LIBRARY=%s" % sysconfig.get_config_var("LIBDIR"),
         ]
 
+        use_seeding = os.environ.get("USE_SEEDING", True)
+        if use_seeding == "OFF":
+            use_seeding = False
+        if not int(use_seeding):
```

Comment:
I think, logically, if disabling seeding is `export USE_SEEDING=OFF`, enabling it back should be `export USE_SEEDING=ON`, which causes an error here when converting to int.

Reply:
Fair point, fixed.

Reply:
great thanks